### ISSUE TITLE
The active keyboard focus is not visible while "Console, iOS, Android and Mac" platform elements are being traversed by the keyboard as shown in the attachments.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#261]
### Steps to Reproduce
1. Launch the Xamarin Workbooks application.
2. "Create a new Workbook" window is getting appeared-> Hit on Tab key to traverse all the interactive elements of the same window.
3. Observe the active keyboard focus on the Console, iOS, Android and Mac" platform elements  as shown in the attachments.

### Expected Behavior
The user should be able to differentiate between the active keyboard focus on these platform elements with the keyboard focus when active keyboard focus is other than those elements.

### Actual Behavior
The active keyboard focus is not visible while "Console, iOS, Android and Mac" platform elements are being traversed by the keyboard as shown in the attachments.

### User Impact
The user won't be able to know the availability of the keyboard focus on these platform elements.
![kb1 1_createnewtab](https://user-images.githubusercontent.com/22956130/39854159-ed7f7578-5443-11e8-893d-fd0f9c2e31f0.png)
![kb1 2_createnewtab](https://user-images.githubusercontent.com/22956130/39854160-eddd608e-5443-11e8-9b88-968c8dc5bca5.png)

### Environment
MAC: macOS High Sierra [10.13.4]
Workbook - 1.4 (master/b3f92f9)

**Comment by WiproPSAccTest** [comment]
Please add the following labels to this bug:
A11yMAS
Accessibility
MAS2.4.7
Xamarin-Workbook-MAC
Wipro

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1813
